.github/workflows/ci.yaml

```diff
@@ -10,7 +10,7 @@ on:
         type: string
         description: 'Base CUDA image:'
         required: false
-        default: 'nvidia/cuda:12.1.1-devel-ubuntu22.04'
+        default: 'NOT SPECIFIED' # use the Dockefile.base default
```

Comment:
The reason I left this default here is to help users quickly find the `nvidia/cuda` container registry and thus locate their preferred image version. If we change it to 'NOT SPECIFIED', users may not be aware of where to find these images.

Reply:
Of course, the problem, as you have correctly identified, is that we now have 2 defaults to manage, and it is easy to forget about updating one of them.

Reply:
Maybe we can fold the "default" into the description

Reply:
> Maybe we can fold the "default" into the description

Good idea

Reply:
Also, 'LATEST' might be better than 'NOT SPECIFIED' as the latter kind of scares me out if I were to call an API with 'unspecified' arguments :grin: